content/spin/v2/key-value-store-tutorial.md

```diff
@@ -176,48 +198,18 @@ command = "tinygo build -target=wasi -gc=leaking -no-debug -o main.wasm main.go"
 
 In this section, we use the Spin SDK to open and persist our application's data inside our default key/value store. This is a special store that every environment running Spin applications will make available for their application. 
 
-> Please note: Spin applications written in Rust can [store and retrieve Rust data structures](./rust-components#storing-data-in-the-spin-key-value-store) in the application's data store.
-
 ### The Spin SDK Version
 
-If you have an existing application and would like to try out the key/value feature, please check the Spin SDK reference in your existing application's configuration. For example, if using Rust please check your application's `Cargo.toml` file. If it refers to version 0.8 or earlier then update the Spin SDK reference to match your upgraded version of Spin:
-
-<!-- @nocpy -->
-
-```toml
-# The Spin SDK.
-spin-sdk = { git = "https://github.com/fermyon/spin", tag = "v0.10.0" }
-```
-
-Similarly an application created using the `http-go` template might need the reference to the Spin SDK in its `go.mod` file updated to look like the following:
-
-<!-- @nocpy -->
-
-```bash
-module github.com/http_go
-
-go 1.17
-
-require github.com/fermyon/spin/sdk/go v0.10.0
-```
-
-The same applies to other programming languages and their respective configuration. This information is provided to prevent you from experiencing an error such as the following:
+If you have an existing application and would like to try out the key/value feature, please check the Spin SDK reference in your existing application's configuration. It is highly recommended to upgrade Spin and the SDK versions to the latest version available.
 
-<!-- @nocpy -->
-
-```bash
-unresolved import spin_sdk::key_value
-key_value::{Error, Store}
-^^^^^^^^^ could not find `key_value` in `spin_sdk`
-```
-
-## Quick Overview - Video
+### Quick Overview - Video
 
 Before we get into the source code, let's watch a quick overview of the new key/value store feature.
 
+<!-- TODO: Does this video need an update? -->
```

Comment:
@tpmccallum Do we need an update for this video?

Reply:
Yes, this will need to be remade to suite v2 if on the v2 page.
I have removed the v1 video content from this v2 page.